### PR TITLE
Maistra 2.3.2

### PR DESCRIPTION
**Pull request by twghu**
Update Maistra proxy to sync with upstream  Istio 1.14 proxy.

### CODE REVIEW

**Comment by twghu** [comment]
WORKSPACE ENVOY_SHA needs to be updated to match Maistra Envoy.

**Comment by twghu** [comment]
Vendor folder to be added

File: common/scripts/setup_env.sh [hunk 1]
```diff
@@ -175,17 +175,14 @@ fi
...
-else
-  LOCAL_OUT="${TARGET_OUT}"
-fi
-
 if [[ "${FOR_BUILD_CONTAINER:-0}" -eq "1" ]]; then
   # Override variables with container specific
   TARGET_OUT=${CONTAINER_TARGET_OUT}
   TARGET_OUT_LINUX=${CONTAINER_TARGET_OUT_LINUX}
-  REPO_ROOT="${WORKDIR}" # /work
+  REPO_ROOT=/work
```
Comment (jwendell) [inline common/scripts/setup_env.sh:182]: This change was introduced by Dario, to make WASM extensions work. Perhaps we should keep it.

**Review by jwendell — COMMENTED** [review]
Perhaps it will be easier to close this PR and create a new one? I'm seeing one commit adding stuff to the vendor dir, whereas these changes come from the other PR...

**Comment by twghu** [comment]
Replaced with https://github.com/maistra/proxy/pull/217

**Pull request closed without merging**